app/components/login_button_component.rb

```diff
@@ -0,0 +1,20 @@
+class LoginButtonComponent < BaseComponent
+  VALID_COLORS = ['primary', 'primary-darker', 'primary-lighter'].freeze
+
+  attr_reader :color, :tag_options
+
+  def initialize(color: 'primary', **tag_options)
+    if !VALID_COLORS.include?(color)
+      raise ArgumentError, "`color` #{color}} is invalid, expected one of #{VALID_COLORS}"
+    end
+
+    @color = color
```

Comment:
To fix the lint error, we'll need to assign `tag_options`

```suggestion
    @color = color
    @tag_options = tag_options
```

Reply:
done, also had to add :big as an attribute since that was coming from ButtonComponent